src/OutputWriters/output_writer_utils.jl

```diff
@@ -9,6 +9,10 @@ using Oceananigans.LagrangianParticleTracking: LagrangianParticles
 ##### Output writer utilities
 #####
 
+convert_to_arch(::CPU, a) = a
+convert_to_arch(::CUDAGPU, a) = CuArray(a)
+convert_to_arch(::ROCMGPU, a) = ROCArray(a)
```

Comment:
This looks identical to `arch_array`, right?

Reply:
Where is `convert_to_arch` used?

Reply:
Not sure. I think it was removed in recent commits on the main branch.

Reply:
Hmm, yeah it was perhaps vestigial then.

Reply:
I searched the codebase and couldn't find any usage of ```convert_to_arch```. I've removed them in the new commits.